packages/office-ui-fabric-react/src/components/Link/Link.Props.ts

```diff
@@ -19,4 +19,8 @@ export interface ILinkProps extends React.HTMLProps<HTMLAnchorElement | HTMLButt
    * Whether the link is disabled
    */
   disabled?: boolean;
+  /**
+ * Optional class for link.
+ */
+  className?: string;
```

Comment:
again, no need for className on this one. It's a standard HTML prop (i.e. if you wanted to pass through data-foo, you wouldn't need to specify it as a property we just pass any standard HTML property though and mix it in)